dataland-frontend/src/components/forms/CreateEuTaxonomyForNonFinancials.vue

```diff
@@ -57,9 +57,11 @@
             <div class="uploadFormSection grid">
               <FormKit type="group" name="data" label="data">
                 <UploadReports
+                  name="UploadReports"
                   ref="UploadReports"
+                  :isEuTaxonomy="true"
                   :referencedReportsForPrefill="templateDataset?.referencedReports"
```

Comment:
Why is this passed as prop here, if it is in fact passed to the children via "provide()" ?